Orange/widgets/evaluate/owpredictions.py

```diff
@@ -76,8 +75,21 @@ class Error(OWWidget.Error):
     score_table = settings.SettingProvider(ScoreTable)
 
     #: List of selected class value indices in the `class_values` list
-    selected_classes = settings.ContextSetting([])
+    PROB_OPTS = ["(None)",
+                 "Classes in data", "Classes known to the model", "Classes in data and model"]
+    PROB_TOOLTIPS = ["Don't show probabilities",
+                     "Show probabilities for classes in the data",
+                     "Show probabilities for classes known to the model,\n"
+                     "including those that don't appear in this data",
+                     "Show probabilities for classes in data that are also\n"
+                     "known to the model"
+                     ]
+    NO_PROBS, DATA_PROBS, MODEL_PROBS, BOTH_PROBS = range(4)
```

Comment:
Fortunately not.

First, the list was composed of all classes that appeared in any of connected predictors. This information is not a part of the `ClassValuesContextHandler` that knows only about the values that appear in the data's class. We can't go from indices in this list to names of values. And even if we could, there wouldn't always be a correct option to choose from.

Second, this never worked anyway because the widget didn't close and open contexts. :)